src/java/picard/sam/MergeSamFiles.java

```diff
@@ -83,6 +90,14 @@
     @Option(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
     public List<String> COMMENT = new ArrayList<String>();
 
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the locations of the positions to merge. "+
+            "Assume bam are sorted and indexed. "+
+            "Reads should not overlap two intervals. "+
```

Comment:
so what do you suggest ? checking that the regions in the bed file are distanst from XX bases ? or checking that each read only overlap one unique region ?